ui/app/routes/vault/cluster/secrets/backend.js

```diff
@@ -7,9 +7,9 @@ export default Ember.Route.extend({
     const { backend } = this.paramsFor(this.routeName);
     const backendModel = this.store.peekRecord('secret-engine', backend);
     const type = backendModel && backendModel.get('type');
-    if (type === 'kv' && backendModel.get('isVersioned')) {
+    if (type === 'kv' && backendModel.get('options.version') === 2) {
       this.get('flashMessages').stickyInfo(
-        `"${backend}" is a versioned kv secrets engine. The Vault UI does not currently support the additional versioning features. All actions taken through the UI in this engine will operate on the most recent version of a secret.`
+        `"${backend}" is a newer version of the KV backend. The Vault UI does not currently support the additional versioning features. All actions taken through the UI in this engine will operate on the most recent version of a secret.`
```

Comment:
I usually try to stay away from "newer" since it will have to be updated as soon as it isn't new, but at that point we're removing this message anyway, right?

Reply:
Yeah, it'd go away once we have support for all of the new stuff.